app/views/admin/link_check_reports/_link_check_report.html.erb

```diff
@@ -15,11 +15,25 @@
     </section>
   <% elsif report.broken_links.present? %>
```

Comment:
I see that you handle links with caution in the loop but I think this implies that if there are only caution links they won't be shown. Is that the intention?